libraries/wutcrt/wut_crt.c

```diff
@@ -2,11 +2,13 @@ void __init_wut_malloc();
 void __init_wut_newlib();
 extern void __init_wut_stdcpp() __attribute__((weak));
 void __init_wut_devoptab();
+void __init_wut_socket();
```

Comment:
These could potentially be weak functions? Could be one way around the AC issue

Reply:
Definitively an option; waiting for more comments on this